client/my-sites/importer/importing-pane.jsx

```diff
@@ -123,6 +123,10 @@ class ImportingPane extends React.PureComponent {
 		);
 	};
 
+	getHeadingTextProcessing = () => {
+		return translate( 'Processing your file. It might take a while, please wait.' );
```

Comment:
Is there anything stopping us from using `this.props.translate` (other than `no-shadow` rules for destructed use)?

Reply:
Ignore for now - we'll tackle this modules use of `translate` in a follow up, so that we're using `this.props.translate` across the whole file 👌